operations/workplace/README.md

```diff
@@ -4,31 +4,31 @@ description: Description of norms on Mattermost Community server
 
 # Workplace
 
-## What is the Mattermost pre-release server?
+## What is the Mattermost Pre-release Server?
 
 The Mattermost pre-release server runs the latest builds of Mattermost's software, including all open source code for Mattermost Team Edition as well as commercial code for Mattermost Enterprise Edition.
 
 It has three objectives:
 
-* Provide a workplace for Mattermost Staff 
-* Test pre-released builds of our core products
-* Engage with our [communities ](../../contributors/contributors/community.md)including contributors and partners 
+* Provide a workplace for Mattermost staff.
+* Test pre-released builds of our core products.
+* Engage with our [communities ](../../contributors/contributors/community.md)including contributors and partners.
 
 **The Mattermost pre-release server is NOT A PRODUCTION SERVER.** It is a live view of work-in-progress and will contain bugs and have occasional downtime.
 
 You can reach the server at:
 
-{% embed url="https://pre-release.mattermost.com" caption="" %}
+{% embed url="https://community.mattermost.com" caption="" %}
 
-The server is also available via the URL [https://community.mattermost.com](https://community.mattermost.com) which receives the same regular pushes of partially tested code as [https://pre-release.mattermost.com](https://pre-release.mattermost.com) and [https://daily-community.mattermost.com](https://daily-community.mattermost.com) which offers daily builds.
+The server is also available via the URL [https://community.mattermost.com](https://community.mattermost.com) which receives the same regular pushes of partially tested code as [https://community-daily.mattermost.com](https://community-daily.mattermost.com) which offers daily builds.
```

Comment:
Since community server is already mentioned above, we can probably change the first link to ``https://community-release.mattermost.com`` and mention that it receives regular pushes of features and/or bug fixes of the next release for testing purposes.

Reply:
Awesome, thanks!